Turbo/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -66,6 +66,7 @@ public static void AddNetworking(this IServiceCollection services)
             // Event Handlers
             services.AddTransient<IEventHandlerManager, EventHandlerManager>();
             services.AddTransient<ITestEventHandler, TestEventHandler>();
+            services.AddTransient<ITestEventHandler, UserLoginEventHandler>();
```

Comment:
should probably use an explicit interface for this event

Reply:
Fixed in latest commit